config/proto/boot_target.yml

```diff
@@ -0,0 +1,8 @@
+version: 1.0
+prototypes:
+  - type: boot_target
+    model: bmc-boot-target
+    manufacturer: Vapor IO
```

Comment:
These manufacturers are going to vary by BMC. Not clear if we should put VaporIO here if this is for production, but it doesn't look like it is.

Reply:
👍  Good point.

These configs aren't for production, the whole repo is still a big ole WIP and still has a bunch of placeholders everywhere.